include/xrpl/protocol/detail/features.macro

```diff
@@ -29,6 +29,8 @@
 // If you add an amendment here, then do not forget to increment `numFeatures`
 // in include/xrpl/protocol/Feature.h.
 
+XRPL_FEATURE(Credentials,                Supported::yes, VoteBehavior::DefaultNo)
+XRPL_FEATURE(AMMClawback,                Supported::yes,  VoteBehavior::DefaultYes)
```

Comment:
I might have missed this earlier, why is AMMClawback default yes vote? @yinyiqian1

Reply:
It should be No.

Reply:
fixed to `No`